README.md

```diff
@@ -41,7 +43,9 @@ or
 @import SplunkOtel;
 ...
 // Your beaconUrl and rumAuth will be provided by your friendly Splunk representative
-[SplunkRum initializeWithBeaconUrl: @"https://rum-ingest.us0.signalfx.com/v1/rum" rumAuth: @"ABCD..." options: nil];
+SplunkRumBuilder *builder = [SplunkRumBuilder withBeaconUrl: @"https://rum-ingest.us0.signalfx.com/v1/rum" rumAuth: @"ABCD..."];
+// Call functions to configure additional options
+[builder build];
 ```
 
 ## Installation options
```

Comment:
How do you set these options now? I'm guessing:

**Swift**

```swift
import SplunkOtel
import SplunkOtelCrashReporting
//..
SplunkRum.initialize(beaconUrl: "https://rum-ingest.<realm>.signalfx.com/v1/rum",
    rumAuth: "<rum-token>",
    options: SplunkRumOptions(environment:"<environment-name>"))
     // Initialize crash reporting module after the iOS agent
SplunkRumCrashReporting.start()
```

**Objective-C**

```objc
@import SplunkOtel;
@import SplunkOtelCrashReporting;
//...
SplunkRumBuilder *builder = [SplunkRumBuilder withBeaconUrl: @"https://rum-ingest.<realm>.signalfx.com/v1/rum" rumAuth: @"<rum-token>"];
// Call functions to configure additional options
     [builder build];
// Initialize crash reporting module after the iOS agent
[SplunkRumCrashReporting start]
```

Reply:
The ObjC snippet is OK?

Reply:
Yes, the ObjC snippet is okay because it is not using the builder not SplunkRum initialize.

Reply:
What about settings like `screenNameSpans` or any other in this list? https://github.com/signalfx/splunk-otel-ios#installation-options

We need to explain how those are set for Swift and ObjC.

Reply:
They all are set on the builder.  For screenNameSpans it would look like this

```swift
builder.screenNameSpans(enabled: true)
```
and the above needs to be called before builder.build().

I see all the options listed in the readMe but no example on how to set them in the options.  All the defaults have remained the same.  Do you want an example like below except expanded to show each option that could be changed?

```swift
SplunkRumBuilder(beaconUrl: "http://127.0.0.1:8989/v1/traces", rumAuth: "FAKE")
        .allowInsecureBeacon(enabled: true)
        .debug(enabled: true)
        .globalAttributes(globalAttributes: ["strKey": "strVal", "intKey": 7, "doubleKey": 1.5, "boolKey": true])
        .deploymentEnvironment(environment: "env")
        .ignoreURLs(ignoreURLs: try! NSRegularExpression(pattern: ".*ignore_this.*"))
        .screenNameSpans(enabled: true)
        .build()
```

Reply:
I think that'll suffice—thanks! :)